tensorflow/python/kernel_tests/nn_ops/BUILD

```diff
@@ -0,0 +1,712 @@
+# Tests of TensorFlow NN kernels written using the Python API.
```

Comment:
This file adds too much. Please check history to see if it was moved since before. The original only adds a single dependency

Reply:
It seems all these changes are coming because of the changes in this commit# - 'https://github.com/tensorflow/tensorflow/commit/ab7c873202574b3cd549a93ccbfd881d659186ca'

Reply:
Yes, so we'd need to edit the files from before this refactoring instead of the new ones.

Reply:
Before this commit, it was renamed to the current file from 'tensorflow/python/kernel_tests/atrous_conv2d_test.py '

Reply:
That's a python file, this comment is in a BUILD file

Reply:
Please review now, I deleted the file.

Reply:
A py file is never renamed to a BUILD file

Reply:
See edits at cl/406745996